docs/source/io/io.rst

```diff
@@ -0,0 +1,26 @@
+.. _io:
+
+Saving and Loading in Librascal
+===============================
+
+The library aims at being able to load/save three types of objects: atomic structures,
+their computed representation and trained machine learning models.
+Many tools already exist for reading atomic srtuctures in python and c++ so we rely on
+ASE_ on the python side and
+support the ASE `.json` atomic structure file format on the c++ side
+(:cpp:class:`AtomicStructure <rascal::AtomicStructure>`). An example is available
+in `json_structure.cc` and `spherical_invariants_example.cc`.
+At the moment we do not support saving representations in native Librascal format
+but on the python side they can be converted to numpy arrays with the `get_features()`
+function and then saved.
```

Comment:
Use case?

Reply:
what are you talking about exactly ?

Reply:
What's the use case for saving representations (rather than saving them as sparsepoints)? Why would anyone want to do this?

Reply:
In ended up saving the feature matrices on many occasions but if I had to name one case it would be as part of a full covariance GPR/KRR model. It should not be saved in sparsepoints because you want to sum over the atoms in the structures of the training set too.

Reply:
Ok, so in that case the "use case" is if you want to save the features, keeping their association with individual structures (which the sparsepoints discards). It seems the easiest workaround would be to save both the feature matrix (using sparsepoints) and a separate array telling you how many atoms go in each structure. Wouldn't you need such an array anyways when extracting the feature matrix with `get_features()`?

Reply:
> Wouldn't you need such an array anyways when extracting the feature matrix with get_features()?

I am not sure about what you mean. Anyhow that's basically the plan of @Luthaf for the new dataholder type that would be used to store representations without a structure manager. This new dataholder will be easy to serialize then.